qemu/tests/cfg/fio_perf.cfg

```diff
@@ -52,9 +52,12 @@
             vhost_nic1 =
             remove_image = no
             Linux:
-                fio_cmd = "i=`/bin/ls /dev/[vs]db` && fio --rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=$i --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --time_based --output=/tmp/fio_result &> /dev/null"
+                virtio_blk:
+                    fio_options = "--rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=/dev/vdb --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --time_based --output=/var/tmp/fio_result &> /dev/null"
+                virtio_scsi:
+                    fio_options = "--rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=/dev/sdb --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --time_based --output=/var/tmp/fio_result &> /dev/null"
```

Comment:
Well, `$i` be removed and use hardcode `sdb/vdb`,  i am not very confident about the stability of this change, the diskname is fixed for all situation ?  @xuwei Could you help review ? Thanks!

Reply:
I think it's OK. Use virtio_blk and virtio_scsi to distinguish the diskname vdb/sdb.
And tingting's codes, just update case \"single_disk.raw_disk.NVMe\" related options, LGTM